azkaban-web-server/src/main/java/cloudflow/servlets/Constants.java

```diff
@@ -9,8 +9,13 @@
   public static final String ATTEMPTS_KEY = "attempts";
   public static final String ID_KEY = "id";
 
+  public static final String FLOW_ID_KEY = "flowId";
+  public static final String FLOW_VERSION_KEY = "flowVersion";
+  public static final String EXPERIMENT_ID_KEY = "experimentId";
+
 
 
   public static final String DEFAULT_500_ERROR_MESSAGE = "Unexpected error encountered.";
```

Comment:
Nit: Not part of your change, it would be nice to add "server" in the message so it is clear to users who are not familiar with HTTP error codes.

Reply:
sure.